### PR TITLE
Adding a quiz page

### LINKED ISSUE #21: coding: create a Quiz
## Description
Create a Quiz by making any new page with the name "Analysis" of five questions on what do Girls feel during Periods. The Questions can have like How is pain or how is blood flow , like that.

### PR DESCRIPTION
**Pull request by anshika208**
#### Description
Issue FIXES #21 
fixes #21 

#### Describe the changes you've made
 i have added a new quiz page in a new few folder named analysis. the quiz page contains navbar and 5 questions which will  come one after the other in a box form.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
'NA'

#### Additional context (OPTIONAL)

quiz1
quiz4
quiz3

#### Checklist

- [x] I have added my name in the contributors list at the end of README.md file.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [ ] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)

### CODE REVIEW

**Comment by anshika208** [comment]
@khushishikhu @Aaishpra i have submitted the pr for adding a quiz page. please have a look!

File: periodo/components/analysis/Analysis.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import React from "react"
+import styles from "./Analysis.module.css"
+
+
+
+
+const Analysis=()=>{
+	const questions = [
+		{
+			questionText: 'Is your periods regular?',
```
Comment (Aaishpra) [inline periodo/components/analysis/Analysis.js:11]: Are your periods regular?

**Review by Aaishpra — CHANGES_REQUESTED** [review]
Just a little grammatical error for now, Will review this PR more precisely once i get free.

File: periodo/components/analysis/Analysis.js (hunk 1 above)
Comment (anshika208) [reply periodo/components/analysis/Analysis.js:11]: sure .. i will look into this... thnku!!

**Review by Aaishpra — APPROVED** [review]
LGTM Loved your work @anshika208 
But from next time try to create PR from different branch

**Pull request merged**